pypeclub/OpenPype
# AfterEffects: Fix rendering for single frame in DL

## Brief description
Single video file ('Asset_renderCompositionMain_v001.avi`) or single frame sequence (`Asset_renderCompositionMain_v001.0000.png`) caused failure when running publishing job in Deadline


## Testing notes:
0. Easiest to test it is to merge this to `feature/OP-2765_AE-to-new-publisher` branch
1. Publish single video (mov or avi - set in Composition Settings in AE) to farm
2. Publish single frame (png) to farm
3. Publish ordinary multi frame sequence to farm 
4. Nothing should fail in `Publish` step of rendering process in DL
